src/plugins/intel_gpu/src/graph/impls/common/loop.cpp

```diff
@@ -37,63 +107,173 @@ struct loop_impl : typed_primitive_impl<loop> {
     void set_node_params(const program_node& arg) override {
         OPENVINO_ASSERT(arg.is_type<loop>());
         const auto& node = arg.as<loop>();
-        _max_iteration = node.get_max_iteration();
         _back_edges = node.get_back_edges();
     }
 
+    void set_body_output_memory(cldnn::network::ptr network, const std::shared_ptr<cldnn::primitive_inst>& inst, memory::ptr mem) const {
```

Comment:
This function name is ambiguous because it is setting not only the network output but also network input and other insts' memory. I think just set_memory_in_body_netwrk is enough.

Reply:
Fixed it. thanks.